README.md

```diff
@@ -36,192 +8,89 @@ After you're finished please remove all the comments and instructions!
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- **Red-green-refactor cycle**<br>TDD involves a cyclical process of writing a failing test (red), writing code to make the test pass (green), and then refactoring the code to improve its design and maintainability.
+- **Incremental development**<br>TDD encourages developers to write code in small increments, testing each piece of functionality as it is added. This helps to catch bugs early on and makes it easier to identify and fix problems.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+Ruby. How to install? Read this [documentation](https://www.ruby-lang.org/en/documentation/installation/).
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
-### Install
-
-Install this project with:
-
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  run git clone git@github.com:harshitajain06/Test-Driven-Development-.git
 ```
---->
-
-### Usage
-
-To run the project, execute the following command:
 
-<!--
-Example command:
+### Install RSpec
 
 ```sh
-  rails server
+  run gem install rspec
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
 ```sh
-
+  rspec solver_spec.rb
 ```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
+👤 **Harshita Jain**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub [link](https://github.com/harshitajain06)
+- Twitter [link](https://twitter.com/harshitajain06)
+- LinkedIn [link](https://www.linkedin.com/in/harshitajain06/)
 
-👤 **Author2**
+👤 **Jonah Wambua**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
```

Comment:
Kindly include a section **Future features** with 1-3 potential features to be implemented in the future.